charts/harvester-load-balancer/values.yaml

```diff
@@ -48,8 +39,10 @@ resources: {}
   #   cpu: 100m
   #   memory: 128Mi
 
-nodeSelector: {}
-
-tolerations: []
-
-affinity: {}
+webhook:
+  replicasCount: 1
+  image:
+    repository: rancher/harvester-load-balancer-webhook
+    pullPolicy: Always
+    tag: "master-head"
+  httpsPort: 8443
```

Comment:
I think we should keep the `nodeSelector, tolerations, and affinity` options. Also, we need to define a proper default resource quota for both LB and webhook.